### PR TITLE
Make voting onymous

### PR DESCRIPTION
**Pull request by dellitsni**
#15

### CODE REVIEW

File: konsent/__init__.py [hunk 1]
```diff
@@ -551,8 +560,24 @@ def list_unions():
...
+    if type == "post":
+        votes = Vote.query.filter(Vote.post_id == id).all()
+    else:
+        votes = Vote.query.filter(Vote.comment_id == id).all()
+
+    # add votes to tuple
+    result = []
+    for vote in votes:
+        author = User.query.filter(User.id == vote.author_id).one()
+        app.logger.info(author)
```
Comment (surrealbytes) [inline konsent/__init__.py]: is that debugging leftover?

**Comment by surrealbytes** [comment]
nice, thank you! all tests pass :+1:

File: konsent/__init__.py [hunk 2]
```diff
@@ -490,8 +494,13 @@ def about():
 @app.route('/members')
 def members():
     union = session['connected_union']
-    return render_template('union-members.html', members=find_members(union))
+    return render_template('union-members.html', members=list_members(union))
 
+# who voted on this post?
+@app.route('/who-voted/<string:type>/<int:id>')
+def who_voted(type, id):
```
Comment (surrealbytes) [inline konsent/__init__.py]: user must be logged in right?

File: konsent/__init__.py (hunk 1 above)
Comment (dellitsni) [reply konsent/__init__.py]: The logging is, yeah. I'll get rid of it, thanks!

File: konsent/__init__.py (hunk 2 above)
Comment (dellitsni) [reply konsent/__init__.py]: That's right, fixed.

**Pull request merged**